pumps/mongo_aggregate.go

```diff
@@ -221,15 +221,18 @@ func (m *MongoAggregatePump) ensureIndexes(c *mgo.Collection) error {
 	}
 
 	var err error
-	ttlIndex := mgo.Index{
-		Key:         []string{"expireAt"},
-		ExpireAfter: 0,
-		Background:  m.dbConf.MongoDBType == StandardMongo,
-	}
+	// CosmosDB does not support "expireAt" option
```

Comment:
@mativm02 can we add that expireAt index is not supported in CosmoDB and it will be skipped in the README / docs?